src/components/MarkdownImage.tsx

```diff
@@ -14,12 +14,16 @@ const MarkdownImage = ({
   width,
   height,
   aspectRatio,
+  src,
   ...rest
 }: MarkdownImageProps) => {
   const imageAspectRatio = parseFloat(aspectRatio)
   let imageWidth = parseFloat(width)
   let imageHeight = parseFloat(height)
 
+  // Replace slashes from windows paths with forward slashes
+  const transformedSrc = src.toString().replace(/\\/g, "/")
```

Comment:
Nice catch @TylerAPfledderer. I think we could do this in [the rehype plugin](https://github.com/ethereum/ethereum-org-fork/blob/dev/src/lib/rehype/rehypeImg.ts) that we've created since there is where we process the MD images.

Reply:
+1. I think this will be good to go with this suggested change @TylerAPfledderer. Thanks for bringing this in.

Reply:
@corwintines @pettinarip Just reviewed this. Do you mean add to this as well as where I originally have this transformation set? It looks like `setImageSize` is currently not being used in the project.

Reply:
@pettinarip @corwintines this has been updated. Move it to the `originalPath` variable in the plugin.